### PR TITLE
moving redis form preview to stable

### PR DESCRIPTION
**Pull request by zikalino**
Preview version was moved to stable.

### CODE REVIEW

**Review by MECHANDR — APPROVED** [review]

**Review by MECHANDR — APPROVED** [review]
`2019-07-01-preview` isn't available, so updating all version references to `2019-07-01`.

**Comment by zikalino** [comment]
avocado errors seem to be unrelated to this pr

File: specification/redis/resource-manager/Microsoft.Cache/preview/2019-07-01/redis.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "RedisManagementClient",
     "description": "REST API for Azure Redis Cache Service.",
-    "version": "2019-07-01-preview"
+    "version": "2019-07-01"
```
Comment (lirenhe) [inline specification/redis/resource-manager/Microsoft.Cache/preview/2019-07-01/redis.json]: Is 2019-07-01 still a preview version? If not, I think you should not put them into the preview folder.

File: specification/redis/resource-manager/readme.md [hunk 2]
```diff
@@ -26,18 +26,19 @@ These are the global settings for the Redis API.
...
 
 
-### Tag: package-preview-2019-07
+### Tag: package-2019-07-preview
 
-These settings apply only when `--tag=package-preview-2019-07` is specified on the command line.
+These settings apply only when `--tag=package-2019-07-preview` is specified on the command line.
 
-```yaml $(tag) == 'package-preview-2019-07'
+```yaml $(tag) == 'package-2019-07-preview'
```
Comment (lirenhe) [inline specification/redis/resource-manager/readme.md]: Just curious, is the api version of  2019-07 still in preview?

Comment (zikalino) [reply specification/redis/resource-manager/readme.md]: yes, the -preview postfix was already removed in the service, but the service team still wants to keep it under "preview" folder, not under stable

**Review by lirenhe — APPROVED** [review]

**Pull request merged**